tests/functional_tests/mining.py

```diff
@@ -143,6 +195,22 @@ def mine(self, via_daemon):
             res = wallet.stop_mining()
         res_status = daemon.mining_status()
         assert res_status.active == False
+        
+    def measure_cpu_power_get_time(self, cores):
+        print("Measuring the currently available CPU power...")
+        time_pi = util_resources.get_time_pi_seconds(cores)
+        print("Time taken to calculate Pi on {} core(s) was {} s.".format(cores, round(time_pi, 2)))
```

Comment:
If you're using format here, you may as well use it to print the value to two digits of precision, rather than rounding.

Reply:
Done.